src/schema/artwork/index.js

```diff
@@ -122,6 +122,88 @@ export const artworkFields = () => {
     category: {
       type: GraphQLString,
     },
+    attribution_class: {
+      type: GraphQLString,
+      description: "Attribution class as classification name",
+      resolve: artwork => {
+        if (artwork.attribution_class) {
+          return capitalizeFirstCharacter(artwork.attribution_class)
+        }
+      },
+    },
+    attribution_class_hover: {
+      type: GraphQLString,
+      description: "Descriptive sentance used as companion for attribution class name display",
```

Comment:
#minor "sentance" &rarr; "sentence", but also maybe it's just a "phrase" not a full sentence